containers/javascript-node-lts-postgres/test-project/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "docker web app",
```

Comment:
This is an invalid name which is causing `npm install` to fail when you F5 the project.

Reply:
@Chuxel Sorry for that! I have updated it now.